vignettes/vitorcid.Rmd

```diff
@@ -20,26 +20,66 @@ knitr::opts_chunk$set(
 library(vitorcid)
 ```
 
-# Overview
-To be added soon.
+# vitorcid
+
+The `vitorcid` package that allows you to generate a CV based on your ORCID profile.
+
+# Installation
+
+Make installation in 3 steps:
```

Comment:
Create a dedicated section from the 1st, 2nd, and 3rd points. Maybe `Authorization`?